packages/contracts-ts/addresses.json

```diff
@@ -0,0 +1,86 @@
+{
+  "AddressManager": {
+    "1": "0xdE1FCfB0851916CA5101820A69b13a4E276bd81F",
+    "5": "0xa6f73589243a6A7a9023b1Fa0651b1d89c177111"
+  },
+  "L1CrossDomainMessenger": {
+    "1": "0x25ace71c97B33Cc4729CF772ae268934F7ab5fA1",
+    "5": "0x5086d1eEF304eb5284A0f6720f79403b4e9bE294"
+  },
+  "L1ERC721Bridge": {
+    "1": "0x5a7749f83b81B301cAb5f48EB8516B986DAef23D",
+    "5": "0x8DD330DdE8D9898d43b4dc840Da27A07dF91b3c9"
+  },
+  "L1StandardBridge": {
+    "1": "0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1",
+    "5": "0x636Af16bf2f682dD3109e60102b8E1A089FedAa8"
+  },
+  "L2OutputOracle": {
+    "1": "0xdfe97868233d1aa22e815a266982f2cf17685a27",
+    "5": "0xE6Dfba0953616Bacab0c9A8ecb3a9BBa77FC15c0"
+  },
+  "OptimismMintableERC20Factory": {
+    "1": "0x4200000000000000000000000000000000000012",
+    "5": "0x4200000000000000000000000000000000000012",
+    "420": "0x4200000000000000000000000000000000000012"
+  },
+  "OptimismPortal": {
+    "1": "0xbEb5Fc579115071764c7423A4f12eDde41f106Ed",
+    "5": "0x5b47E1A08Ea6d985D6649300584e6722Ec4B1383"
+  },
+  "PortalSender": {
+    "1": "0x0A893d9576b9cFD9EF78595963dc973238E78210",
+    "5": "0xe7FACd39531ee3C313330E93B4d7a8B8A3c84Aa4"
+  },
+  "ProxyAdmin": {
+    "1": "0x4200000000000000000000000000000000000018",
+    "5": "0x4200000000000000000000000000000000000018"
+  },
+  "SystemConfig": {
+    "1": "0x229047fed2591dbec1eF1118d64F7aF3dB9EB290",
+    "5": "0xAe851f927Ee40dE99aaBb7461C00f9622ab91d60"
+  },
+  "SystemDictator": {
+    "1": "0xB4453CEb33d2e67FA244A24acf2E50CEF31F53cB"
+  },
+  "SystemDictator_goerli": {
```

Comment:
This seperation here between SystemDictator and SystemDictator_goerli happens whenever the code gen detects the abis don't match between networks